operator/Dockerfile

```diff
@@ -20,10 +16,6 @@ COPY go.sum go.sum
 # Add GitHub hosts as known ones so that git does not complain.
 RUN mkdir -p -m 0700 ~/.ssh && ssh-keyscan github.com >> ~/.ssh/known_hosts
```

Comment:
I wonder whether we even use git+ssh now that the `insteadOf` below is gone and the repo is public...
Not in scope of this PR, but neither is the below change ;-)

Reply:
Removed it